1227-customer_registry/574-application_administrator_opens_a_customer_institution.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Application Administrator opens a Customer Institution
+
+    @574
+    Scenario Outline: Application Administrator opens a Customer Institution
+        Given that the user is logged in as Application Administrator
+        When they open a Customer Institution
+        Then they see fields:
+            | Name in organization registry |
+            | Display name                  |
+            | Short display name            |
+            | Archive name                  |
+            | Feide Organization ID         |
+        And they see the Save button
+        And they see the list of current Institution Administrators
```

Comment:
skulle dette være med i visningen allikevel, @terjehell ? Var det ikke sånn at admin-lista ikke skulle være på denne siden? men kun på `Brukere`-siden?

Reply:
Denne storyen er for App admin, og App admin kan ikke se users-siden, så man må kunne legge til Institution admin for eksisterende Customer Institutions her
(Men den skal ikke vises i Min Institusjon, nei)

Reply:
merk at "Min institusjon" (Institution admin) og "Administrer institusjoner" (App admin) ikke er det samme lenger